media/transcoder/src/test/java/com/example/transcoder/CreateJobTemplateTest.java

```diff
@@ -70,7 +70,11 @@ public void beforeTest() throws IOException {
     TEMPLATE_NAME =
         String.format(
             "projects/%s/locations/%s/jobTemplates/%s", PROJECT_NUMBER, LOCATION, TEMPLATE_ID);
-    DeleteJobTemplate.deleteJobTemplate(PROJECT_ID, LOCATION, TEMPLATE_ID);
+    try {
+      DeleteJobTemplate.deleteJobTemplate(PROJECT_ID, LOCATION, TEMPLATE_ID);
+    } catch (Exception e) {
+      // Don't worry if the template doesn't already exist.
```

Comment:
Same as above. I don't think we should swallow this error in our tests.

Reply:
Please see above.